### ISSUE TITLE
kit cmd error

### ISSUE THREAD

**Issue by zwell** [kujtimiihoxha/kit#21]
I install the kit follow the doc, but when I try "kit help", the error happen. 
ERRO[0000] The project must be in the $GOPATH/src folder for the generator to work.

**Comment by 0x457** [comment]
I got the same error:
![image](https://user-images.githubusercontent.com/5813610/50436416-c62dbb00-0920-11e9-8791-a42645aa524e.png)

**Comment by pandanrain** [comment]
go version 1.11 
`pandanrain:elk pandanrain$ echo $GOPATH

/www/workspace/go/

pandanrain:elk pandanrain$ pwd

/www/workspace/go/src/elk

pandanrain:elk pandanrain$ kit help

ERRO[0000] The project must be in the $GOPATH/src folder for the generator to work. `

as below  cd $GOPATH/src/child-dir not work for me

**Comment by jfiuza** [comment]
Actually i think the problem might be GOPATH. If it ends with a / you probably need to remove it

**Comment by navono** [comment]
If use go mod, the project directory no need to be in $GOPATH/src directory and also have the problem:
```
?[31mERRO?[0m[0000] The project must be in the $GOPATH/src folder for the generator to work.
```

**Comment by maeglindeveloper** [comment]
@kujtimiihoxha , is there a way to handle this :) ?
Thanks :)

**Comment by herrjemand** [comment]
Same issues here

**Comment by kujtimiihoxha** [comment]
`kit` does not support `go mod` for now, I do not know if I will have the time to implement this functionality in the near future.

**Comment by Sakurasan** [comment]
![](https://i.loli.net/2019/02/27/5c7622fa35dde.png)

**Comment by kujtimiihoxha** [comment]
@Sakurasan @navono @herrjemand  The project needs to be in a child dir of `GOPATH/src` something like `GOPATH/src/my_project`, for now `go mod` is not supported but I will see how much it takes to modify `kit`.

@jfiuza that should be fixed with #31

Closing for now.
